packages/block-editor/src/components/block-compare/index.js

```diff
@@ -23,8 +23,8 @@ class BlockCompare extends Component {
 
 		return difference.map( ( item, pos ) => {
 			const classes = classnames( {
-				'editor-block-compare__added': item.added,
-				'editor-block-compare__removed': item.removed,
+				'editor-block-compare__added block-editor-block-compare__added': item.added,
```

Comment:
Of my own curiosity, I did confirm that `classnames` is capable of handling the assigning of two classes with respect to a single boolean property:

```
⇒ node       
> var cn = require( 'classnames' )
undefined
> cn( { 'foo bar': true } )
'foo bar'
```